circle.yml

```diff
@@ -1,3 +1,21 @@
-test:
+general:
+  artifacts:
+    - "tmp"
+
+machine:
+  node:
+    version: 6.1.0
+
+dependencies:
+  post:
+    - npm install -g accesslint-cli
+
+compile:
   override:
     - bundle exec middleman build --verbose
+
+test:
+  override:
```

Comment:
Could you explain why this is `override:` and not `post:`, as described in the accesslint-ci README?